client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/utils.ts

```diff
@@ -1,14 +1,17 @@
-const sourceSiteId = 174455321; // dotcompatterns
+import { PATTERN_SOURCE_SITE_ID } from './constants';
+
 const patternPreviewUrl =
 	'https://public-api.wordpress.com/wpcom/v2/block-previews/pattern?stylesheet=pub/blank-canvas&pattern_id=';
 
-const getPatternPreviewUrl = ( id: number ) => `${ patternPreviewUrl }${ id }-${ sourceSiteId }`;
+export const encodePatternId = ( patternId: number ) =>
+	`${ patternId }-${ PATTERN_SOURCE_SITE_ID }`;
+
+export const getPatternPreviewUrl = ( id: number ) =>
```

Comment:
Do we need to handle locale here?

Reply:
I thought another [PR](https://github.com/Automattic/wp-calypso/pull/67478) had a refactor on it, so it might be better to consider the locale there.